### DISCUSSION TITLE
Required files when developig with TABLEBROWSER

### DISCUSSION THREAD

**Discussion by pamoroso** [Interlisp/medley#1362] [Q&A] [answered]
According to the [Lisp Library Modules](http://www.bitsavers.org/pdf/xerox/interlisp-d/198809_Medley_1.0/300006_Lisp_Library_Modules_Sep88.pdf) manual (page 283 of the PDF), when developing with `TABLEBROWSER` it is necessary to load also the source of the declarations file `TABLEBROWSERDECLS`. However, when trying to load the file I get the error `File TABLEBROWSER not found`:

![tberror](https://github.com/Interlisp/medley/assets/10342479/f6f23cc1-2740-4a4b-a03f-3dee11097dcb)

And trying to create aNY `TABLEBROWSER` related record, for example `TABLEITEM`, yelds the error `undefined record name`:

```
7_ (create TABLEBROWSER)
 undefined record name
 at   TABLEBROWSER    in    (create TABLEBROWSER)
Can't CLISP expand expression (create TABLEBROWSER).
```

I see that `TABLEBROWSERDECLS` was moved to `obsolete/library` in the source tree. Is the file still required?

**Comment by nbriggs** [comment, marked as answer]
@masinter changed this in August 2022 so you just `(LOADCOMP TABLEBROWSER)` if you want to develop TABLEBROWSER code.  See if that works for you.

The perils of updating the code without updating the documentation :-(

**Comment by pamoroso** [comment]
`(LOADCOMP 'TABLEBROWSER)` (with quoted file name) works as expected, thanks.

**Comment by masinter** [comment]
I will raise a documentation issue
